dev/statement_serializer.h

```diff
@@ -1787,23 +1788,22 @@ namespace sqlite_orm {
                 subCtx.use_parentheses = true;
 
                 std::stringstream ss;
-                if (!is_compound_operator<T>::value) {
+                if constexpr (!is_compound_operator<T>::value) {
                     if (!sel.highest_level && context.use_parentheses) {
                         ss << "(";
                     }
                     ss << "SELECT ";
-                    call_if_constexpr<is_rowset_deduplicator_v<return_type>>(
+                    if constexpr (is_rowset_deduplicator_v<return_type>) {
                         // note: make use of implicit to-string conversion
-                        [&ss](std::string keyword) {
-                            ss << keyword << ' ';
-                        },
-                        sel.col);
+                        std::string keyword = sel.col;
```

Comment:
do we have `std::string` copying here?

Reply:
Yes, see comment above, which didn't change.